core/src/components/back-button/test/a11y/back-button.e2e.ts

```diff
@@ -1,6 +1,32 @@
+import AxeBuilder from '@axe-core/playwright';
 import { expect } from '@playwright/test';
 import { configs, test } from '@utils/test/playwright';
 
+/**
+ * Only ios mode uses ion-color() for the back button
+ */
```

Comment:
Should we also remove these since the latest change provides more info on the scenario?

Reply:
We can. My reasoning was that we often put a comment explaining why we're only testing in one mode or with one direction, even if it's the default reason (behavior doesn't differ between modes/directions). So I figured it made sense to have a comment here, especially since it's a different reason than normal. But I'd be fine with removing it.

Reply:
Got it, I'm okay with either or.